src/Nncase.Passes/Rules/Neutral/CombineQuantize.cs

```diff
@@ -50,7 +50,24 @@ public sealed partial class CombineQuantizeConcat : RewriteRule<Pattern>
             {
                 if (userAnalysis[e].Count() > 1)
                 {
-                    return null;
+                    foreach (var user in userAnalysis[e])
+                    {
+                        if (user is Call { Target: Nncase.IR.Math.Quantize } userCall)
+                        {
+                            var quantUser = userCall.Arguments[Nncase.IR.Math.Quantize.QuantParam.Index];
+                            if (quantUser != quantParam)
+                            {
+                                return null;
+                            }
+                        }
+                        else
+                        {
+                            if (user is not Nncase.IR.Tuple)
+                            {
```

Comment:
should consider pattern of densenet

Reply:
I have add the condition about this Scenes. please help to check it again.